source/frontend/src/components/common/mapFSM/MapStateMachineContext.tsx

```diff
@@ -151,17 +160,18 @@ export const MapStateMachineProvider: React.FC<React.PropsWithChildren<unknown>>
             latLng: event.featureSelected.latlng,
             pimsPropertyId: event.featureSelected?.pimsLocationFeature?.PROPERTY_ID ?? null,
           });
+          // TODO: verify that this is still needed
```

Comment:
Do we have a Jira for this? I'm leaning towards associating TODOs to something - since we don't have a practice in place to periodically check for these and clean them up.